moveit_ros/hybrid_planning/CMakeLists.txt

```diff
@@ -68,25 +68,35 @@ rclcpp_components_register_nodes(moveit_hybrid_planning_manager "moveit_hybrid_p
 rclcpp_components_register_nodes(moveit_global_planner_component "moveit_hybrid_planning::GlobalPlannerComponent")
 rclcpp_components_register_nodes(moveit_local_planner_component "moveit_hybrid_planning::LocalPlannerComponent")
 
-install(TARGETS ${LIBRARIES} hybrid_planning_test_node
+# Install libraries
+install(TARGETS ${LIBRARIES}
+  EXPORT export_${PROJECT_NAME}
   ARCHIVE DESTINATION lib
   LIBRARY DESTINATION lib
-  RUNTIME DESTINATION lib/${PROJECT_NAME}
+  RUNTIME DESTINATION bin
   INCLUDES DESTINATION include)
 
+# Install binary
+install(TARGETS hybrid_planning_test_node
+  ARCHIVE DESTINATION lib
+  LIBRARY DESTINATION lib
+  RUNTIME DESTINATION bin
+  DESTINATION lib/${PROJECT_NAME}
+)
+
```

Comment:
You can convert this into one install command like this (@JafarAbdi showed me this):

```c++
install(
  TARGETS 
    ${LIBRARIES}
    hybrid_planning_test_node
  EXPORT ${PROJECT_NAME}Targets
  ARCHIVE DESTINATION lib
  LIBRARY DESTINATION lib
  RUNTIME DESTINATION lib/${PROJECT_NAME}
  INCLUDES DESTINATION include
)
```